### PR TITLE
SK-1516-TRANS-WEB-Email-configure_mail_apple 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md [hunk 1]
```diff
@@ -34,11 +39,44 @@ Es posible configurar sus cuentas MX Plan en el cliente de correo que usted util
...
 
-| | |
-|---|---|
-|![mailmac](images/mail-mac-mxplan01.png){.thumbnail}|Seleccione `Otra cuenta de correo`{.action} y haga clic en `Cuentas de correo`{.action}.|
-|Introduzca en la ventana "**Añadir cuenta de correo**" la siguiente información: <br>- el **nombre** de su cuenta de correo <br>- Su **dirección de correo electrónico** <br>- La **contraseña** de su dirección de correo electrónico |![mailmac](images/mail-mac-mxplan02.png){.thumbnail}|
-|![mailmac](images/mail-mac-mxplan03.png){.thumbnail}|Introduzca la información en la siguiente ventana: <br>- Deje su **dirección de correo electrónico** ya introducida <br>- Introduzca su dirección de correo electrónico completa en el **Nombre de usuario** <br>- Deje su **contraseña** ya introducida <br>\- Seleccione `POP` o `IMAP` (recomendado) en **Tipo de cuenta**<br>- Introduzca `ssl0.ovh.net` en el **servidor de recepción** <br>- Introduzca también `ssl0.ovh.net` en el **servidor de envío**<br><br>Para finalizar la configuración, haga clic en `Conectar`{.action}|
+Siga los pasos de instalación haciendo clic en las fichas siguientes:
+
+> [!tabs]
+> **Paso 1**
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md]:
```suggestion
> **Etapa 1**
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md [hunk 2]
```diff
@@ -34,11 +39,44 @@ Es posible configurar sus cuentas MX Plan en el cliente de correo que usted util
...
+>>
+>> Seleccione `Otra cuenta Mail`{.action} y haga clic en `Cuenta Mail`{.action}.<br><br>
+>> ![mailmac](images/mail-mac-email01.png){.thumbnail .w-400 .h-600}
+>>
+> **Paso 2**
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md]:
```suggestion
> **Etapa 2**
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md [hunk 3]
```diff
@@ -34,11 +39,44 @@ Es posible configurar sus cuentas MX Plan en el cliente de correo que usted util
...
+>>
+>> Introduzca la siguiente información en la ventana "**Añadir una cuenta de correo**": <br><br>
+>> - Un **Nombre** para su cuenta de correo
+>> - Su **Dirección de correo electrónico**
+>> - El **Contraseña** de su dirección de correo<br>
+>> ![mailmac](images/mail-mac-email02.png){.thumbnail .w-400 .h-600}
+>>
+> **Paso 3**
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md]:
```suggestion
> **Etapa 3**
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md [hunk 4]
```diff
@@ -1,11 +1,17 @@
 ---
 title: 'MX Plan - Configure su cuenta de correo electrónico en Mail para macOS'
-updated: 2024-10-01
+excerpt: Cómo configurar una cuenta MX Plan en Mail de macOS
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md]:
```suggestion
excerpt: Descubra cómo configurar una cuenta MX Plan en Mail de macOS
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md [hunk 5]
```diff
@@ -50,45 +88,104 @@ OVHcloud también ofrece una aplicación web que permite acceder a su dirección
...
 
-[Configurar una cuenta Email Pro en Mail de macOS](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_mail_macos)
+[MX Plan - Configurar una cuenta de correo electrónico en Mail para iPhone e iPad](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_ios)
+
+[Email Pro - Configurar una cuenta de correo electrónico en Mail para macOS](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_mail_macos)<br>
+[Email Pro - Configurar una cuenta de correo electrónico en Mail para iPhone e iPad](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_ios)
+
+[Exchange - Configurar una cuenta de correo electrónico en Mail de macOS](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_ios)<br>
+[Exchange - Configurar una cuenta de correo electrónico en Mail para iPhone e iPad](pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_mail_macos/guide.fr-fr.md)
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.es-es.md]: why guide.fr-fr.md at the end of the link ?

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.it-it.md [hunk 6]
```diff
@@ -1,11 +1,17 @@
 ---
 title: MX Plan - Configura il tuo account di posta elettronica su Mail per macOS
-updated: 2024-10-01
+excerpt: Scopri come configurare il tuo indirizzo email MX Plan su Mail di macOS
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.it-it.md:3]:
```suggestion
excerpt: Questa guida ti mostra come configurare il tuo indirizzo email MX Plan su Mail di macOS
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.it-it.md [hunk 7]
```diff
@@ -50,41 +88,104 @@ OVHcloud propone anche un'applicazione Web che permette di accedere al tuo indir
...
 > Per informazioni sulla configurazione di un indirizzo email dall’applicazione Mail su macOS, consulta [il centro assistenza Apple](https://support.apple.com/it-it/guide/mail/mail35803/mac).
 
-Contatta la nostra Community di utenti all’indirizzo <https://community.ovh.com/en/>.
+[MX Plan - Configurare un account email su Mail per iPhone e iPad](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_ios)
+
+[Email Pro - Configurare un account email su Mail per macOS](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_mail_macos)<br>
+[Email Pro - Configurare un account email su Mail per iPhone e iPad](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_ios)
+
+[Exchange - Configurare un account email su Mail di macOS](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_ios)<br>
+[Exchange - Configurare un account email su Mail per iPhone e iPad](pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_mail_macos/guide.fr-fr.md)
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.it-it.md]: guide.fr-fr.md ?

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md [hunk 8]
```diff
@@ -50,41 +79,104 @@ OVHcloud oferuje również aplikację internetową umożliwiającą korzystanie
...
 
-|Typ serwera|Nazwa serwera|Metoda szyfrowania|Port|
-|---|---|---|---|
-|Wchodzący(IMAP)|ssl0.ovh.net|SSL/TLS|993|
-|Wychodzący(SMTP)|ssl0.ovh.net|SSL/TLS|465|
+|Informacje|Opis|
+|---|---|
+|Nazwa użytkownika|Wpisz pełny adres e-mail **complete**|
+|Hasło|Wpisz hasło wybrane dla tego konta e-mail|
+|Serwer **EUROPE** (ruch przychodzący)|imap.mail.ovh.net **ou** ssl0.ovh.net|
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md]: **ou** => I think in PL "ou" translated by "lub"

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md [hunk 9]
```diff
@@ -50,41 +79,104 @@ OVHcloud oferuje również aplikację internetową umożliwiającą korzystanie
...
 
-![mailmac](images/mail-mac-mxplan04.png){.thumbnail}
+- **Konfiguracja POP**
 
-### Dodatkowe informacje
+|Informacje|Opis|
+|---|---|
+|Nazwa użytkownika|Wpisz pełny adres e-mail **complete**|
+|Hasło|Wpisz hasło wybrane dla tego konta e-mail|
+|Serwer **EUROPE** (ruch przychodzący)|pop.mail.ovh.net **ou** ssl0.ovh.net|
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md]: **ou** => I think in PL "ou" translated by "lub"

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md [hunk 10]
```diff
@@ -50,41 +79,104 @@ OVHcloud oferuje również aplikację internetową umożliwiającą korzystanie
...
-|---|---|---|---|
-|Wchodzący(POP)|ssl0.ovh.net|SSL/TLS|995|
-|Wychodzący(SMTP)|ssl0.ovh.net|SSL/TLS|465|
+- **Konfiguracja SMTP**
+
+|Informacje|Opis|
+|---|---|
+|Nazwa użytkownika|Wpisz pełny adres e-mail **complete**|
+|Hasło|Wpisz hasło wybrane dla tego konta e-mail|
+|Serwer **EUROPE** (ruch wychodzący)|smtp.mail.ovh.net **ou** ssl0.ovh.net|
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md]: **ou** => I think in PL "ou" translated by "lub"

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md [hunk 11]
```diff
@@ -50,41 +79,104 @@ OVHcloud oferuje również aplikację internetową umożliwiającą korzystanie
...
+> Aby uzyskać więcej informacji na temat konfigurowania konta e-mail z poziomu aplikacji Mail na macOS, skorzystaj z [Centrum pomocy Apple](https://support.apple.com/pl/pomoc/mail35803/mac).
+
+[MX Plan - Konfiguracja konta e-mail w aplikacji Mail na urządzeniach iPhone i iPad](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_ios)
+
+[E-mail Pro - Konfiguracja konta e-mail w aplikacji Mail na urządzeniach z systemem macOS](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_mail_macos)<br>
+[E-mail Pro - Konfiguracja konta e-mail w aplikacji Mail na urządzeniach iPhone i iPad](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_ios)
+
+[Exchange - Konfiguracja konta e-mail w aplikacji Mail na urządzeniach z systemem macOS](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_ios)<br>
+[Exchange - Konfiguracja konta e-mail w aplikacji Mail na urządzeniach iPhone i iPad](strony/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_mail_macos/guide.fr-en.md)
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pl-pl.md]: guide.fr-en.md ?

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pt-pt.md [hunk 12]
```diff
@@ -50,41 +79,104 @@ A OVHcloud também disponibiliza uma aplicação web que permite aceder ao seu e
...
 
-|Tipo de servidor|Nome do servidor|Método de encriptação|Porta|
-|---|---|---|---|
-|Entrando(POP)|pro**?**.mail.ovh.net (a menção **"?"** é substituída pelo número do seu servidor)|SSL/TLS|995|
-|Saída (SMTP)|pro**?**.mail.ovh.net (a menção **"?"** é substituída pelo número do seu servidor)|SSL/TLS|465|
+- **Configuração SMTP**
+
+|Informação|Descrição|
+|---|---|
+|Nome de utilizador|Introduza o endereço de e-mail **completo*|
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pt-pt.md]:
```suggestion
|Nome de utilizador|Introduza o endereço de e-mail **completo**|
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pt-pt.md [hunk 13]
```diff
@@ -50,41 +79,104 @@ A OVHcloud também disponibiliza uma aplicação web que permite aceder ao seu e
...
-No caso de uma configuração em **IMAP**, os valores são os seguintes:
+- **Para uma configuração em IMAP**
 
-|Tipo de servidor|Nome do servidor|Método de encriptação|Porta|
-|---|---|---|---|
-|Entrando (IMAP)|pro**?**.mail.ovh.net (a menção **"?"** é substituída pelo número do seu servidor)|SSL/TLS|993|
-|Saída (SMTP)|pro**?**.mail.ovh.net (a menção **"?"** é substituída pelo número do seu servidor)|SSL/TLS|465|
+|Informação|Descrição|
+|---|---|
+|Nome de utilizador|Introduza o endereço de e-mail **completo*|
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pt-pt.md]:
```suggestion
|Nome de utilizador|Introduza o endereço de e-mail **completo**|
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pt-pt.md [hunk 14]
```diff
@@ -50,41 +79,104 @@ A OVHcloud também disponibiliza uma aplicação web que permite aceder ao seu e
...
 > Para obter mais informações sobre a configuração de um endereço de e-mail a partir da aplicação Mail no macOS, consulte [Centro de Ajuda da Apple](https://support.apple.com/pt-pt/guide/mail/mail35803/mac).
 
-Fale com a nossa comunidade de utilizadores: <https://community.ovh.com/en/>.
+[MX Plan - Configurar a sua conta de e-mail no Mail para iPhone e iPad](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_ios)
+
+[E-mail Pro - Configurar a sua conta de e-mail no Mail para macOS](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_mail_macos)<br>
+[E-mail Pro - Configurar a sua conta de e-mail no Mail para o iPhone e o iPad](/pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_ios)
+
+[Exchange - Configurar a sua conta de e-mail no Mail do macOS](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_ios)<br>
+[Exchange - Configurar a sua conta de e-mail no Mail para o iPhone e o iPad](/pages/web_cloud/email_and_collaborative_solutions/microsoft_exchange/how_to_configure_mail_macos/guide.pt-pt.md)
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.pt-pt.md]: guide.pt-pt.md ?

**Review by Kazarus4 — COMMENTED** [review]
review done for me

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**